backends/ent/annotations.go

```diff
@@ -39,6 +40,28 @@ func (backend *Backend) AddAnnotations(documentID, name string, values ...string
 	return backend.withTx(backend.saveAnnotations(data...))
 }
 
+// AddNodeAnnotations applies multiple named annotation values to a single node.
+func (backend *Backend) AddNodeAnnotations(nodeID, name string, values ...string) error {
+	data := ent.Annotations{}
+
+	nodeUUID, err := backend.client.Node.Query().
+		Where(node.IDEQ(nodeID)).
+		OnlyID(backend.ctx)
+	if err != nil {
+		return fmt.Errorf("querying nodes: %w", err)
+	}
+
+	for _, value := range values {
+		data = append(data, &ent.Annotation{
+			NodeID: nodeUUID,
+			Name:   name,
+			Value:  value,
+		})
+	}
+
+	return backend.withTx(backend.saveAnnotations(data...))
```

Comment:
For the `Node` schema, this isn't a UUID, just a string currently. So, I think we can skip the query and just attempt to apply the annotations, which assumes the nodes were previously inserted.

```suggestion
	for _, value := range values {
		data = append(data, &ent.Annotation{
			NodeID: nodeID,
			Name:   name,
			Value:  value,
		})
	}

	return backend.withTx(backend.saveAnnotations(data...))
```

Reply:
This applies to all other instances of querying for a "nodeUUID" as well

Reply:
Done!